tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -952,6 +960,32 @@ callable. This can be used to pass functions as arguments.
           the ``()`` operator directly. This behavior is implemented to avoid
           performance issues on direct function calls.
 
+Lambda functions
+^^^^^^^^^^^^^^^^
+
+Lambda functions allow you to declare functions that do not belong to a class. Instead a Callable object is created and assigned to a variable directly.
+This can be useful to create Callables to pass around without polluting the class scope.
+
+::
+
+    var lambda = func(x): print(x)
+    lambda.call(42) # Prints "42"
+
+Lambdas can be given a name for debugging purposes::
```

Comment:
Nitpick but its shorter, for the same meaning, feel free to deny this specific comment
```
Lambdas can be named for debugging purposes
```